oeis.py

```diff
@@ -217,6 +217,36 @@ def A000041(n):
     return len(partitions(n))
 
 
+def Fibonacci(n): 
+    if n<0: 
+        print("Incorrect input") 
+    elif n==1: 
+        return 0
+    elif n==2: 
+        return 1
+    else: 
+        return Fibonacci(n-1)+Fibonacci(n-2)
+
+def A000045():
+    args = parse_args()
```

Comment:
The call to parse_args is already done in the `main()` function, it's not needed to call it again.